src/wrapper/java_vm/vm.rs

```diff
@@ -40,9 +43,47 @@ use InitArgs;
 ///
 /// ## Launching JVM from Rust
 ///
-/// To [launch][launch-vm] a JVM from a native process, enable the `invocation` feature.
+/// To [launch][launch-vm] a JVM from a native process, enable the `invocation` feature
+/// in the Cargo.toml:
+///
+/// ```toml
+/// jni = { version = "0.12.3", features = ["invocation"] }
+/// ```
+///
 /// The application will require linking to the dynamic `jvm` library, which is distributed
-/// with the JVM.
+/// with the JVM, and allow to use `JavaVM#new`:
+///
+/// ```rust
```

Comment:
I made it work with `invocation` feature enabled (see 7c79ee2), but if you run just `cargo test` it will naturally fail. Is it possible to modify it (in a simple way) so that it is ignored if `invocation` is absent, but runs with it?

Reply:
https://stackoverflow.com/a/50314936 This seems like a solution

Reply:
Thanks, I haven't tried to apply it to the whole block, will check if it works.

Reply:
It works, the only thing is that IDEA does not handle it fully correctly:

![image](https://user-images.githubusercontent.com/7531824/59782174-0ebda400-92c6-11e9-96de-c500be99593c.png)